lib/elixir/lib/kernel.ex

```diff
@@ -3712,12 +3712,14 @@ defmodule Kernel do
         raise ArgumentError, "expected to: to be given as argument"
 
       for fun <- List.wrap(funs) do
-        {name, args, as, as_args} = Kernel.Utils.defdelegate(fun, opts)
```

Comment:
You can do it with a single comprehension:

    for fun <- List.wrap(funs),
        {name, args, as, as_args} <- Kernel.Utils.defdelegate(fun, opts) do

:)